src/zenml/stack_stores/models/rest_api_models.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from pydantic import BaseModel
+
+
+class Version(BaseModel):
```

Comment:
I'm a bit confused about this file - how is the Version a rest_api_model?

Reply:
Good point, this model is no longer needed now that the service does not return its version (this was a wrapper to send a version over the network)